client/packages/programs/src/JsonForms/components/Prescription/StockLineTable.tsx

```diff
@@ -34,32 +36,51 @@ export const StockLineTable = ({
 
   const columns = useColumns<DraftStockOutLine>(
     [
+      {
+        width: 60,
+        key: 'expiry',
+        label: 'label.expiry',
+        accessor: ({ rowData }) => rowData.expiryDate,
+        Cell: ExpiryDateCell,
+        align: ColumnAlign.Right,
+      },
       {
         width: '100px',
         key: 'batch',
         label: 'label.batch',
         accessor: ({ rowData }) => rowData.stockLine?.batch,
       },
-      {
-        width: '100px',
-        key: 'expiry',
-        label: 'label.expiry',
-        accessor: ({ rowData }) => rowData.expiryDate,
-      },
       {
         width: '90px',
         key: 'available',
         label: 'label.available-units',
         accessor: ({ rowData }) =>
-          NumUtils.round(rowData.stockLine?.availableNumberOfPacks ?? 0, 2),
+          NumUtils.round(
+            (rowData.stockLine?.availableNumberOfPacks ?? 0) *
+              (rowData.stockLine?.packSize ?? 1),
+            2
+          ),
       },
       {
         width: '55px',
-        key: 'numberOfPacks',
+        key: 'unitQuantity',
         label: 'label.units-issued',
-        accessor: ({ rowData }) => rowData.numberOfPacks,
-        setter: handleUpdateDraft,
-        Cell: PackQuantityCell,
+        accessor: ({ rowData }) =>
+          NumUtils.round(
+            (rowData.numberOfPacks ?? 0) * (rowData.packSize ?? 1),
+            3
+          ),
+        setter: row => {
```

Comment:
I could avoid using the `as` by keeping the key as `numberOfPacks`, but this would mean temporarily storing the unit count in the `numberOfPacks` field, then updating it by multiplying. But I think it's bad practise to put a value into an incorrectly named field, even if only temporarily. This is a bit hacky, but I think it's less misleading than the alternative.

Reply:
I think its probably better make that a comment in the actual code rather than here, I agree though :)

Reply:
Sure: 2f70cf0991